gravitee-apim-console-webui/src/services-ngx/portal-page-content.service.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Inject, Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { Constants } from '../entities/Constants';
+import { NewPortalPageContent, PortalPageContent } from '../entities/management-api-v2';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PortalPageContentService {
+  constructor(
+    private http: HttpClient,
+    @Inject(Constants) private readonly constants: Constants,
+  ) {}
+
+  public getPageContent(contentId: string): Observable<PortalPageContent> {
+    return this.http.get<PortalPageContent>(`${this.constants.env.baseURL}/portal-page-content/${contentId}`);
+  }
+
+  public createPageContent(newPortalPageContent: NewPortalPageContent): Observable<PortalPageContent> {
+    return this.http.post<PortalPageContent>(`${this.constants.env.baseURL}/portal-page-content`, newPortalPageContent);
```

Comment:
I think it should be in plural for the creation

```suggestion
    return this.http.post<PortalPageContent>(`${this.constants.env.baseURL}/portal-page-contents`, newPortalPageContent);

Reply:
Done!